docs/index.md

```diff
@@ -5,42 +5,37 @@ nav_order: 1
 
 # Welcome
 
-Santa is a binary authorization system for macOS. Here you will find the
-documentation for understanding how Santa works, how to deploy it and how to
-contribute.
+Santa is a binary authorization system for macOS. It consists of a system extension that monitors for executions, a daemon that makes execution decisions based on the contents of a local database, a GUI agent that notifies the user in case of a block decision, and a command-line utility for managing the system and synchronizing the database with a server.
 
-#### Introduction
+It is named Santa because it keeps track of binaries that are naughty or nice.
 
-The following documents give an overview of how Santa accomplishes binary
-authorization at the enterprise scale.
+Here you'll find documentation for [understanding](introduction/index.md) how Santa works, [deploying Santa](deployment/index.md), and how to [contribute](development/contributing.md) to the project. 
```

Comment:
Let me know what you think: I took out that whole sentence, since there is a whole overview of the documentation lower on the page. And I extended the title.